public_html/pipeline_health.php

```diff
@@ -687,6 +734,7 @@ function get_gh_team_repos($team){
   'branch_dev_code_owner_reviews' => 'dev: code owner reviews',
   'branch_dev_required_num_reviews' => 'dev: 1 review',
   'branch_dev_enforce_admins' => 'dev: enforce admins',
+  'branch_template_restrict_push' => 'T push',
```

Comment:
```suggestion
  'branch_template_restrict_push' => 'TEMPLATE: restrict push',
```
Any reason to keep it so short?

Reply:
Yeah, I wrote that originally but it makes the column a lot wider, compressing all of the other columns a lot as a result.

The header isn't so important any more to be honest as we have so many repos that it usually scrolls out of view quite quickly. Hovering over any cells in the column shows the full description so it's easy to see what the failures are.